Src/AutoFixtureUnitTest/EmailAddressLocalPartTest.cs

```diff
@@ -0,0 +1,169 @@
+            Assert.Throws<ArgumentNullException>(
+                () => new EmailAddressLocalPart(null));
+            // Teardown
+        }
+
+        [Theory]
+        [InlineData(@"@CantStartWithAtSign")]
+        [InlineData(@".StartsWithPeriod")]
+        [InlineData(@"Has..TwoPeriods")]
+        [InlineData(@"EndsWithPeriod.")]
+        [InlineData(@".")]
+        [InlineData(@"Ima Notquoted")]
+        public void InitializeWithInvalidLocalPartThrows(string localPart)
+        {
+            // Fixture setup
+            // Exercise system and verify outcome
+            Assert.Throws<ArgumentException>(
+                () => new EmailAddressLocalPart(localPart));
+            // Teardown
+        }
+
+        [Theory]        
+        [InlineData(@"""Quoted\\WithSlash""")]
+        [InlineData("\"Quoted\\\rWithLineBreak\"")]
+        [InlineData(@"""Both\""Quoted""")]
+        [InlineData(@"service/department")]
+        [InlineData(@"$StartWithDollar")]
+        [InlineData(@"!def!xyz%abc")]
+        [InlineData(@"_Good.Email")]
+        [InlineData(@"~")]
+        [InlineData(@"""Quoted@AtSign""")]
+        [InlineData(@"Ima.InternalPeriod")]
+        [InlineData(@"""Ima.Quoted""")]
+        [InlineData(@"""Ima Quoted""")]
+        public void InitializeWithValidLocalPartSetsLocalPart(string localPart)
+        {
+            // Fixture setup            
+            var sut = new EmailAddressLocalPart(localPart);
+            // Exercise system
+            string result = sut.LocalPart;
+            // Verify outcome
+            Assert.Equal(localPart, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void ToStringReturnsCorrectResult()
+        {
+            // Fixture setup
+            string expected = "good.localPart";
+            var sut = new EmailAddressLocalPart("good.localPart");
+            // Exercise system
+            var result = sut.ToString();
+            // Verify outcome
+            Assert.Equal(expected, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutIsEquatable()
+        {
+            // Fixture setup
+            // Exercise system
+            var sut = new EmailAddressLocalPart("good.localPart");
+            // Verify outcome
+            Assert.IsAssignableFrom<IEquatable<EmailAddressLocalPart>>(sut);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullObject()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            object other = null;
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullSut()
+        {
+            // Fixture setup
+            var sut = new EmailAddressLocalPart("good.localPart");
+            UriScheme other = null;
```

Comment:
Was actually an oversight, `UriScheme` should have been `EmailAddressLocalPart` there.  Even with that change though, I don't think both tests are needed as they both exercise the same code path since the conditional within `Equals` is false regardless of the type if the object is set to null.  The other case (where a non-null non-`EmailAddressLocalPart` instance is sent to `Equals`) is also already covered, by `SutDoesNotEqualAnonymousObject` so I'm just going to remove this particular test.